modules/core/src/main/java/org/locationtech/jts/geom/impl/PackedCoordinateSequence.java

```diff
@@ -271,14 +300,14 @@ public Double(Coordinate[] coordinates, int dimension) {
      * @param coordinates
      */
     public Double(Coordinate[] coordinates) {
-      this(coordinates, 3);
+      this(coordinates, 3,0);
```

Comment:
(Picky) space after the comma.

Reply:
aside: I could not find any code formatting guidelines for the project, I setup some defaults that kind of matched in my IDE an hoped for the best.